Prog/Calculate_equity_gaps.do

```diff
@@ -0,0 +1,57 @@
+* Program: Calculate_equity_gaps
+* Created by: Yipeng Su
+* Last modified: 10/22/2018
+
```

Comment:
add a display statement to write the date to the log.  let me know if you need code.